test/Fail/Issue2423.err

```diff
@@ -1,3 +1,3 @@
 Issue2423.agda:24,1-17
-Cannot eliminate type  S  with projection  R.f
+Cannot eliminate type  S  with projection  S._.f
```

Comment:
Different disambiguation than before.
Issue2423.agda says:
```
-- NOW:
-- Cannot eliminate type  S  with projection  R.f

-- Shows the chosen disambiguation in error message.
```
Which is no longer in sync with the error message.